README.md
[![GitHub release](https://img.shields.io/github/v/release/glasnonck/BingoLists.svg)](https://github.com/glasnonck/BingoLists/releases/latest)
[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/glasnonck/BingoLists/master/LICENSE)

# BingoLists
JavaScript lists of bingo items.


changelog.txt
KotOR Bingo Changelog v6.0

Overall Features of v6.0 (a.k.a. TL;DR):
- All 25 groups have 9 tasks each for 225 total tasks
- Added 41 tasks and removed 22 tasks since v5.0
- 81 returning tasks changed category
- 29 returning tasks were tweaked
- Removed several near-duplicate tasks to encourage more variety in rows
- The hardest tasks have been fully revamped to remove the truly awful ones and add some additional Star Forge and Lehon tasks
- Introduced tiered tasks with multiple versions; for instance, 8 Different Lightsaber Power Crystals, 13 Different Lightsaber Power Crystals, and 15 Different Lightsaber Power Crystals
- GenoHaradan tasks have been pared down and grouped together for fewer rows that require them
- Tasks that only required a certain level have been entirely removed; the Yuthura XP glitch made them both difficult to balance and boring
- Tasks involving Vitality, Force Points, and Saves now require those to be Permanent, i.e. not including timed buffs like Force Valor or stims
- Some rows may now require both the Leviathan and either all 5 Star Maps or Lehon

Added Tasks:
- Defeat Deadeye Duncan to 1
- Persuade a Prospective Sith to Leave to 1
- Complete "A Desert Ambush" to 3
- Kill the Lavin Thugs to 3
- Obtain 4 Complete Area Maps to 3
- Help Three Taris Bounties Escape to 7
- Three Permanent Immunities to 9
- Master Toughness and Master Conditioning to 9
- Clear the Taris Sith Base to 10
- 8 Different Lightsaber Power Crystals to 10
- 120 Permanent Maximum Vitality to 11
- 150 Permanent Maximum Force Points to 12
- 10 Thermal Detonators to 12
- Reedem 2 Sith to the Light to 13
- 12 Permanent Points in all Saves to 13
- Obtain 7 Complete Area Maps to 14
- Complete "Jagi's Challenge" to 14
- 20 Completed Quests in Journal to 15
- Kill 12 Mandalorians to 15
- Maximum Permanent Immunities to 15
- All 5 Advanced Droid Weapons to 17
- 30 Completed Quests in Journal to 18
- ICE Breaker to 19
- 8 Different Jedi Robes to 19
- Complete All 4 Minigame Quests to 19
- Kill 20 Mandalorians to 20
- Niklos Accuses You of Cheating to 20
- Clear the Prison Block with Party Member to 20
- All 4 Disguises to 21
- Kill Keeper Orsaa to 21
- Sapith and Upari Crystals to 22
- Obtain 10 Complete Area Maps to 23
- All 9 Verpine Equipments to 24
- Darth Revan's Robes to 24
- Star Forge Robes to 24
- Kill Bastila to 24
- 11 Different Jedi Robes to 25
- All 10 Heavy Armors to 25
- Clear the Rakatan Temple to 25
- Kill Four Party Members to 25
- Redeem 6 Sith to the Light to 25

Removed Tasks:
- Force Storm from 4
- Complete "Mission for the Republic" from 9
- Kill Zuulan from 11
- Three Viper Kinrath Corpses from 12
- Fully Upgraded Bendak's Blaster from 12
- Download 5 Area Schematics from 12
- Kill Vorn from 13
- Kill Rulan from 13
- Master Scoundrel's Luck and Master Sense from 14
- Three Unique Bowcasters from 17
- 40 Points in All Skills from 18
- Achieve Level 20 from 18
- Heart of the Guardian from 21
- Adrenaline Stimulator from 21
- Complete "Premium Merchant" from 21
- Kill the Duros on Lehon from 22
- Level 8 in Two Classes from 24
- Kill 40 Unique Enemies from 24
- Level 10 in Base Class from 25
- Complete All 9 Companion Sidequests from 25
- Clear All 4 Sith Strongholds from 25
- Light Side Mastery from 25

Changed Tasks:
- In 3: Above 30 Net Stealth Skill to 20 Net Stealth Skill
- In 4: Solve the Rakatan Riddles to Escape the Mind Prison
- In 5: Sneak Attack IX to Improved Scoundrel's Luck
- In 9: Dark Side Mastery to Light or Dark Side Mastery
- In 10: Romance Bastila to Kiss Bastila
- In 11: GenoHaradan Poison Blade to Force Choke Lorgal
- In 14: Four Third-Tier Force Powers to Three Third-Tier Force Powers
- In 16: 250 Maximum Vitality to 150 Permanent Maximum Vitality
- In 16: Complete All Dantooine Quests to Complete All 8 Dantooine Quests
- In 17: 250 Maximum Force Points to 180 Permanent Maximum Force Points
- In 17: Complete All Manaan Quests to Complete All 7 Manaan Quests
- In 18: At Least 25 in All Saves to 16 Permanent Points in All Saves
- In 18: Complete All Kashyyyk Quests to Complete All 7 Kashyyyk Quests
- In 19: Complete All Tatooine Quests to Complete All 11 Tatooine Quests
- In 20: Clear Both Sith Bases to Clear the Manaan Sith Base
- In 20: Complete All Korriban Quests to Complete All 13 Korriban Quests
- In 22: All 7 Different Lightsaber Colors to 7 Different Lightsaber Colors
- In 23: Clear the Warrior Rakatan Compound to Clear a Rakatan Settlement
- In 25: All 15 Lightsaber Power Crystals to 15 Different Lightsaber Power Crystals

Changed and Moved Tasks:
- Complete "Unfinished Business" from 6 to Give Motta the Mysterious Box in 7
- 20 Gaffi Sticks from 7 to Complete "Sand People" in 6
- One of Each Pazaak Card from 11 to All 18 Pazaak Sidedeck Cards in 10
- Fully Upgraded Bendak's Blaster from 12 to Fully Upgraded Zaalbar's Bowcaster in 9
- One of Each Deadly Mine in 15 to Four Different Deadly Mines in 18
- Resolve All 3 Murder Cases in 16 to Complete All 3 Murder Quests in 17
- 13 Unique Lightsaber Power Crystals in 18 to 13 Different Lightsaber Power Crystals in 20
- Resolve All Party Family Issues in 20 to Complete All 4 Party Family Quests in 21
- Complete All Taris Quests in 20 to Complete All 18 Taris Quests in 23
- Kill At Least 25 Mandalorians in 25 to Kill 25 Mandalorians in 23

Moved Tasks:
- Kill Calo Nord from 1 to 2
- Defend the Sandcrawler Miners from 3 to 4
- Complete "Pazaak Rules" from 3 to 4
- Clear the Endar Spire from 3 to 9
- Recruit HK-47 from 4 to 5
- Kill a Hive Kinrath from 4 to 5
- Kill a Rancor from 5 to 4
- Kill Kandon Ark from 5 to 6
- Complete "Rakghoul Serum" from 5 to 6
- Sith Armor from 6 to 4
- Qel-Droma Robes from 6 to 5
- 10,000 Credits from 7 to 6
- Return Sasha to Dantooine from 8 to 10
- Carbonite Projector Mark II from 9 to 8
- Firkrann Crystal from 9 to 8
- Strand Zaalbar on Kashyyyk from 9 to 10
- Win All 8 Swoop Races from 9 to 14
- Purchase a Store's Entire Inventory from 10 to 7

kotor-bingo.js
var bingoListVersion = "6.0";
var bingoList = [];

bingoList[1] = [
  { name: "Kill Selven", types: ["Taris","TarQuest","Bounty"] },
  { name: "Fully Upgraded Republic Mod Armor", types: ["Upgrade"] },
  { name: "Rescue Tanis", types: ["Tatooine"] },
  { name: "Czerka Corp. Resource ID", types: ["Kashyyyk"] },
  { name: "Kill the Whimpering Locker", types: ["Manaan"] },
  { name: "Implant Level 3", types: ["Character"] },
  { name: "Discover Zelka Forn's Secret", types: ["Taris"] },
  { name: "Defeat Deadeye Duncan", types: ["Taris", "Duel"] },
  { name: "Persuade a Prospective Sith to Leave", types: ["Korriban"] },
];
bingoList[2] = [
  { name: "Blow Up Matrik's Apartment", types: ["Taris", "Bounty"] },
  { name: "Fully Upgraded Echani Fiber Armor", types: ["Upgrade"] },
  { name: "Sell Sharina's Wraid Plate", types: ["Tatooine"] },
  { name: "Destroy Droid (Force Power)", types: ["Force power", "Character"] },
  { name: "Bondar Crystal", types: ["Crystal"] },
  { name: "Exactly 0 Credits", types: ["Credits"] },
  { name: "Attend Yun Genda's Party", types: ["Disguise", "Taris", "TarQuest", "Gender"] },
  { name: "Kill Calo Nord", types: ["Bosses", "Heavy"] },
  { name: "Sith Medallion", types: ["Korriban"] },
];
bingoList[3] = [
  { name: "Get Dia's Bounty Lifted", types: ["Taris","Bounty"] },
  { name: "Talk to Samnt", types: ["Dantooine"] },
  { name: "Complete \"A Desert Ambush\"", types: ["Tatooine"] },
  { name: "20 Net Stealth Skill", types: ["Character"] },
  { name: "Obtain 4 Complete Area Maps", types: ["Maps"] },
  { name: "Kill the Lavin Thugs", types: ["Taris"] },
  { name: "Rubat Crystal", types: ["Crystal"] },
  { name: "Poison Both Yuthura and Uthar", types: ["Korriban"] },
  { name: "Help with Lyn Sekla's Audition", types: ["Taris"] },
];
bingoList[4] = [
  { name: "Complete \"Pazaak Rules\"", types: ["Pazaak"] },
  { name: "Damind Crystal", types: ["Crystal"] },
  { name: "Convict Both Rickard and Handon", types: ["Dantooine","Murder"] },
  { name: "Rescue the Mandalorian on Korriban", types: ["Korriban"] },
  { name: "Defend the Sandcrawler Miners", types: ["Tatooine"] },
  { name: "Complete \"The Trouble with Gizka\"", types: [] },
  { name: "Escape the Mind Prison", types: ["Riddles"] },
  { name: "Sith Armor", types: ["Taris", "Disguise"] },
  { name: "Kill a Rancor", types: ["Taris", "Rancor"] },  
];
bingoList[5] = [
  { name: "Kill a Shyrack Alpha", types: ["Korriban"] },  
  { name: "Opila Crystal", types: ["Crystal", "Opila"] },
  { name: "Kill a Hive Kinrath", types: ["Dantooine"] },
  { name: "15 Tach Glands", types: ["Kashyyyk"] },
  { name: "Qel-Droma Robes", types: ["Korriban", "Robes", "Hunt", "AlignedItem"] },
  { name: "Recruit HK-47", types: ["Party", "Tatooine", "HK"] },  
  { name: "Improved Scoundrel's Luck", types: ["Character"] },
  { name: "Uncanny Dodge 2", types: ["Character"] },
  { name: "Win 4 Fighter Skirmishes", types: [] },
];
bingoList[6] = [
  { name: "10,000 Credits", types: ["Credits"] },
  { name: "Complete \"Rakghoul Serum\"", types: ["Taris","TarQuest"] },  
  { name: "Kill Kandon Ark", types: ["Taris"] },
  { name: "Become Sector Champion on Manaan", types: ["Manaan", "ManSwoops", "Swoops"] },
  { name: "Eralam Crystal", types: ["Crystal"] },
  { name: "Two Weapon Specializations", types: ["Character"] },
  { name: "5 Different Lightsaber Colors", types: ["Colors"] },
  { name: "Complete \"Sand People\"", types: ["Tatooine"] },
  { name: "Successfully Interrogate the Sith Prisoner", types: ["Manaan"] },
];
bingoList[7] = [
  { name: "Davik's War Suit", types: ["Taris","Bosses"] },
  { name: "150 Racing Bonds", types: ["Tatooine", "Swoops", "TatSwoops"] },
  { name: "Complete \"Sandral-Matale Feud\"", types: ["Dantooine"] },
  { name: "Give Motta the Mysterious Box", types: ["Riddles"] },
  { name: "Jenruax Crystal", types: ["Crystal"] },
  { name: "Complete \"Honest Debt\"", types: ["Kashyyyk"] },
  { name: "Nextor Crystal", types: ["Crystal"] },
  { name: "Purchase a Store's Entire Inventory", types: ["Credits"] },
  { name: "Help Three Taris Bounties Escape", types: ["Bounty", "Taris"] },
];
bingoList[8] = [
  { name: "Free Hudrow", types: ["Taris"] },
  { name: "Clear the Hidden Bek Base", types: ["Clear", "Taris"] },
  { name: "Kill an Albino Kath Hound", types: ["Dantooine"] },
  { name: "Defeat Furko Nellis's Tournament Deck", types: ["Pazaak", "Tatooine"] },
  { name: "Solari Crystal", types: ["Crystal", "Korriban", "Hunt", "AlignedItem"] },
  { name: "Fully Repair HK-47", types: ["Party", "HK"] },
  { name: "Kill Darth Bandon", types: ["Bosses"] },  
  { name: "Carbonite Projector Mark II", types: ["Korriban", "Tombs"] },
  { name: "Firkrann Crystal", types: ["Crystal","Korriban"] },
];
bingoList[9] = [
  { name: "Clear the Endar Spire", types: ["Clear"] },
  { name: "Fully Upgraded Mission's Vibroblade", types: ["Mission", "Upgrade"] },
  { name: "Kill Kel Algwinn", types: ["Korriban"] }, 
  { name: "Light or Dark Side Mastery", types: ["Alignment", "Mastery", "FP"] },
  { name: "Kill Belaya", types: ["Belaya","Juhani","Party"] },  
  { name: "Get Banned From Manaan", types: ["Manaan"] }, 
  { name: "Master Toughness and Master Conditioning", types: ["Character"] }, 
  { name: "Fully Upgraded Zaalbar's Bowcaster", types: ["Zaalbar", "Upgrade"] },
  { name: "Three Permanent Immunities", types: ["Immunity"] },
];
bingoList[10] = [
  { name: "Clear the Taris Sith Base", types: ["Clear", "Taris"] },
  { name: "Two Krayt Dragon Pearls", types: ["Tatooine", "Crystal"] },
  { name: "Redeem Ajunta Pall's Spirit", types: ["Korriban", "Tombs", "Redeem"] },
  { name: "8 Different Lightsaber Power Crystals", types: ["Crystal"] },
  { name: "Return Sasha to Dantooine", types: ["Party", "Zaalbar"] },
  { name: "Kiss Bastila", types: ["Party", "Gender"] },
  { name: "Strand Zaalbar on Kashyyyk", types: ["Kashyyyk","Zaalbar","Party","Strand"] },
  { name: "A Droid with Upgrade Class 3", types: ["Character", "HK", "T3", "Party"] },
  { name: "All 18 Pazaak Sidedeck Cards", types: ["Pazaak"] }, 
];
bingoList[11] = [
  { name: "Talk to Komad on Kashyyyk", types: ["Tatooine", "Kashyyyk"] }, 
  { name: "Force Choke Lorgal", types: ["Manaan", "Geno"] },
  { name: "GenoHaradan Blaster", types: ["Dantooine", "Geno", "Blasters"] },  
  { name: "120 Permanent Maximum Vitality", types: ["Character"] },
  { name: "Defeat Twitch", types: ["Taris", "Duel", "Bounty"] },
  { name: "Solve the Light Ring Puzzle", types: ["Korriban", "Tombs"] },
  { name: "100,000 Credits", types: ["Credits"] },
  { name: "All Master-Level Blaster Feats", types: ["Character"] },
  { name: "Phond Crystal", types: ["Crystal", "Korriban", "Tombs"] },
];
bingoList[12] = [
  { name: "10 Thermal Detonators", types: ["Thermals", "Mika", "Dark"] },
  { name: "GenoHaradan Power Gloves", types: ["Manaan", "Geno", "Gloves"] },  
  { name: "GenoHaradan Stealth Unit", types: ["Tatooine", "Geno", "Belts"] },
  { name: "GenoHaradan Visor", types: ["Kashyyyk", "Geno"] },
  { name: "Two Bendak's Blasters", types: ["Taris", "Duel", "Bounty", "Blasters"] },
  { name: "Master Caution, Empathy, and Gear Head", types: ["Character"] },
  { name: "150 Permanent Maximum Force Points", types: ["Character", "FP"] },
  { name: "Circlet of Saresh", types: ["Kashyyyk","Hunt","AlignedItem"] },
  { name: "Naga Sadow's Poison Blade", types: ["Korriban","Tombs"] },
];
bingoList[13] = [
  { name: "Complete \"The Promised Land\"", types: ["Taris","TarQuest"] },
  { name: "Clear the Vulkar Base", types: ["Taris", "Clear"] },
  { name: "Redeem 2 Sith to the Light", types: ["Redeem", "Korriban", "Family", "Tombs"] },
  { name: "Complete \"Mandalorian Raiders\"", types: ["Dantooine", "Mandos"] },
  { name: "12 Permanent Points in All Saves", types: ["Character"] },
  { name: "All Three Force Immunity Feats", types: ["Character", "Immunity"] },  
  { name: "Both Holocrons", types: ["Tatooine","Korriban","Family"] },
  { name: "Have Sunry Declared Innocent", types: ["Kashyyyk","Manaan","Murder","Party","Jolee"] },
  { name: "Defeat Both Defense Droids", types: ["Kashyyyk"] }, 
];
bingoList[14] = [
  { name: "Three Third-Tier Force Powers", types: ["Character"] },
  { name: "Kill Jorak Uln", types: ["Korriban", "Tombs", "Redeem", "Opila"] },
  { name: "Token From Tortured Selkath", types: ["Manaan"] }, 
  { name: "Fully Upgraded Armor, Blaster, and Sword", types: ["Upgrade"] },
  { name: "Obtain 7 Complete Area Maps", types: ["Maps"] },
  { name: "Win All 8 Swoop Races", types: ["Swoops", "ManSwoops", "TatSwoops"] },
  { name: "Kill Griff", types: ["Mission", "Tatooine", "Party", "Family"] },
  { name: "Recruit All Party Members", types: ["Party","Mission","Zaalbar","Juhani","Jolee","HK","T3"] },  
  { name: "Complete \"Jagi's Challenge\"", types: ["Party"] },
];
bingoList[15] = [
  { name: "Complete All 5 Taris Bounty Quests", types: ["Taris", "Bounty"] },
  { name: "Complete \"Hidden Hunters\"", types: ["Kashyyyk","Mandos"] },
  { name: "20 Completed Quests in Journal", types: ["Quests", "QuestNum"] },
  { name: "Complete \"Chieftain in Need\"", types: ["Kashyyyk", "Party", "Chieftain", "Strand"] },
  { name: "Complete \"Finding Dustil\"", types: ["Party"] },
  { name: "Talk to Yuthura on Dantooine", types: ["Korriban","Redeem"] }, 
  { name: "Moisture Vaporators", types: ["Tatooine","HK"] },
  { name: "Kill 12 Mandalorians", types: ["Mandos"] },
  { name: "Maximum Permanent Immunities", types: ["Immunity"] },
];
bingoList[16] = [
  { name: "Clear Davik's Estate", types: ["Taris", "Clear"] },
  { name: "Level 6 in Two Classes", types: ["Character"] },
  { name: "150 Permanent Maximum Vitality", types: ["Character"] },
  { name: "Complete All 8 Dantooine Quests", types: ["Dantooine","Quests"] },
  { name: "Fully Upgraded Baragwin Assault Blade", types: ["Upgrade"] },
  { name: "Both Exoskeletons", types: ["Credits"] },
  { name: "All 3 Great Hunt Journals", types: ["Hunt"] },
  { name: "Kill Four Terentateks", types: ["Hunt"] },
  { name: "All 4 Sith Lord Artifacts", types: ["Tombs", "Korriban", "AlignedItem"] },
];
bingoList[17] = [
  { name: "Clear the Hrakert Rift Station", types: ["Manaan", "Clear"] },
  { name: "Complete All 3 Murder Quests", types: ["Murder", "Mastery"] },
  { name: "180 Permanent Maximum Force Points", types: ["Character", "FP"] },
  { name: "Complete All 7 Manaan Quests", types: ["Manaan","Quests","ManSwoops"] },
  { name: "Complete \"Threat from Xor\"", types: ["Juhani", "Party"] },
  { name: "All 5 Advanced Droid Weapons", types: ["Tombs"] },
  { name: "Zaalbar Wielding Bacca's Ceremonial Blade", types: ["Zaalbar","Kashyyyk","Chieftain"] },
  { name: "Raito's Gaderffii", types: ["Tatooine"] },
  { name: "Give Griff a Tach Gland", types: ["Tatooine", "Mission"] },
];
bingoList[18] = [
  { name: "Clear the Sand People Enclave", types: ["Tatooine", "Clear"] },  
  { name: "Luxum Crystal", types: ["Crystal", "Manaan", "Robes"] },
  { name: "Complete All 7 Kashyyyk Quests", types: ["Kashyyyk", "Quests", "Strand", "Mastery"] },
  { name: "16 Permanent Points in All Saves", types: ["Character"] },
  { name: "Redeem 4 Sith to the Light", types: ["Korriban", "Tombs", "Redeem"] },
  { name: "30 Completed Quests in Journal", types: ["QuestNum", "Quests"] },
  { name: "Kill Hulas", types: ["Geno"] },
  { name: "Four Different Deadly Mines", types: ["Character"] },
  { name: "21 Base Charisma", types: ["Character"] },
];
bingoList[19] = [
  { name: "Clear the Shadowlands", types: ["Kashyyyk", "Clear"] },
  { name: "8 Different Jedi Robes", types: ["Robes"] },
  { name: "Verpine Prototype Shield", types: ["Armband", "Verpine", "Dark", "Leviathan", "Mika"] },
  { name: "ICE Breaker", types: ["Leviathan", "Maps"] },
  { name: "Five Alignment-Restricted Items", types: ["Tombs", "AlignedItem"] },
  { name: "Three Light Alignment-Restricted Items", types: ["AlignedItem"] },
  { name: "Complete All 11 Tatooine Quests", types: ["Tatooine", "Quests", "TatSwoops"] },
  { name: "10 Prestige Points", types: ["Korriban"] },
  { name: "Complete All 4 Minigame Quests", types: ["Duel", "Swoops", "Pazaak"] },
];
bingoList[20] = [
  { name: "13 Different Lightsaber Power Crystals", types: ["Crystal"] },
  { name: "Kill 20 Mandalorians", types: ["Mandos"] },
  { name: "Clear the Sith Academy", types: ["Korriban", "Clear"] },
  { name: "Clear the Manaan Sith Base", types: ["Manaan", "Clear"] },
  { name: "Niklos Accuses You of Cheating", types: ["Pazaak"] },
  { name: "Ship Parts", types: ["StarMaps"] },
  { name: "All 5 Star Maps", types: ["StarMaps"] },
  { name: "Complete All 13 Korriban Quests", types: ["Korriban", "Quests"] },
  { name: "Clear the Prison Block with Party Member", types: ["Leviathan"] },
];
bingoList[21] = [
  { name: "Kill Saul Karath", types: ["Bosses", "Leviathan"] },
  { name: "All 4 Disguises", types: ["Leviathan", "Disguise"] },
  { name: "Complete All 4 Party Family Quests", types: ["Party", "Mission", "Zaalbar", "Family"] },
  { name: "Complete All 11 Bounty Quests", types: ["QuestNum", "Geno", "Bounty", "Duel"] },  
  { name: "Mantle of the Force", types: ["Credits", "StarMaps", "Thermals", "Colors"] },
  { name: "Dominator Gauntlets", types: ["Credits", "Mika", "Gloves", "StarMaps"] },
  { name: "Kill The One", types: ["StarMaps", "Rancor"] },
  { name: "Kill Keeper Orsaa", types: ["StarMaps"] },
  { name: "All 10 Armbands", types: ["Armband", "Mika", "StarMaps"] },
];
bingoList[22] = [
  { name: "Defeat Malak on the Leviathan", types: ["Leviathan"] },
  { name: "Kill Calo, Davik, Bandon, and Saul", types: ["Leviathan", "Bosses"] },
  { name: "All 14 Medium Armors", types: ["Credits", "Mika"] },
  { name: "Kill the Trandoshan Smugglers", types: ["Mika", "StarMaps", "Thermals"] }, 
  { name: "7 Different Lightsaber Colors", types: ["Colors", "Thermals", "Credits", "StarMaps"] },
  { name: "Verpine Zal Alloy Mesh", types: ["Mandos", "Verpine", "StarMaps"] },
  { name: "Sith Power Gauntlets", types: ["Gloves", "StarMaps", "Thermals", "Verpine"] },
  { name: "Sapith and Upari Crystals", types: ["Crystal", "StarMaps", "Thermals", "Verpine"] },
  { name: "Kill Jolee and Juhani", types: ["StarMaps", "Dark"] },
];
bingoList[23] = [
  { name: "Obtain 10 Complete Area Maps", types: ["Maps"] },
  { name: "All 14 Light Armors", types: ["Credits", "Geno"] },
  { name: "Complete \"Rakatan Research\"", types: ["StarMaps", "Thermals"] },
  { name: "Clear a Rakatan Settlement", types: ["Clear", "StarMaps"] },
  { name: "Kill 25 Mandalorians", types: ["Mandos", "StarMaps", "Verpine"] },  
  { name: "Complete All 18 Taris Quests", types: ["Bounty", "Duel", "TarQuest"] },
  { name: "Star Map: Kashyyyk as Revan", types: ["Kashyyyk", "StarMaps"] },
  { name: "All 6 Stealth Generators", types: ["Credits", "StarMaps", "Belts"] },
  { name: "Reach the Star Forge", types: ["StarMaps", "Dark"] },
];
bingoList[24] = [
  { name: "All 23 Blaster Pistols", types: ["Credits", "Blasters"] },
  { name: "Clear the Leviathan", types: ["Leviathan", "Clear"] }, 
  { name: "Defeat Every Pazaak Player Once", types: ["Pazaak"] },
  { name: "All 9 Verpine Equipments", types: ["Verpine", "Mika", "StarMaps", "Mandos"] },
  { name: "Darth Revan's Robes", types: ["AlignedItem", "StarMaps", "Dark"] },
  { name: "Star Forge Robes", types: ["AlignedItem", "StarMaps"] },
  { name: "40 Completed Quests in Journal", types: ["Quests", "QuestNum"] },
  { name: "Romance Carth", types: ["Party", "StarMaps", "Leviathan", "Gender"] },
  { name: "Kill Bastila", types: ["StarMaps"] },
];
bingoList[25] = [
  { name: "Get Suvam Tam's Pazaak Discount", types: ["Pazaak"] }, 
  { name: "15 Different Lightsaber Power Crystals", types: ["Crystal", "StarMaps", "Verpine"] },
  { name: "11 Different Jedi Robes", types: ["Robes", "Verpine", "StarMaps", "Dark", "AlignedItem"] },
  { name: "All 10 Heavy Armors", types: ["StarMaps", "Heavy", "Mika", "Credits"] },
  { name: "All 12 Gloves", types: ["Gloves", "Mika", "StarMaps", "Thermals", "Verpine"] },
  { name: "Clear the Rakatan Temple", types: ["Clear", "Dark", "Verpine", "StarMaps"] },
  { name: "Kill Four Party Members", types: ["Belaya", "Zaalbar", "Mission", "Dark", "StarMaps"] },
  { name: "Force Zaalbar to Kill Mission", types: ["Zaalbar", "Mission", "Dark", "StarMaps"] },
  { name: "Redeem 6 Sith to the Light", types: ["Redeem", "StarMaps"] },
];

$(function () { bingo(bingoList, 5, bingoListVersion); });
